lib/config/options/index.ts

```diff
@@ -2044,6 +2044,13 @@ const options: RenovateOptions[] = [
     stage: 'repository',
     globalOnly: true,
   },
+  {
+    name: 'writeDiscoveredRepos',
+    description: 'Writes discovered repositories to a file and then exit.',
```

Comment:
```suggestion
    description: 'Writes discovered repositories to a JSON file and then exit.',
```

Do we mean "any file" or do we mean "only JSON file"???

If we only support JSON file output, apply the suggestion.

Reply:
Yes, currently there is only a JSON file option.